2015/proposals/banaszkiewicz-piotr-amy.md

```diff
@@ -0,0 +1,214 @@
+# Enhance Amy, a workshop-management platform for Software Carpentry
+
+## Abstract
+
+The number of Software Carpentry's workshops run weekly dynamically grows. All
```

Comment:
On Tue, Mar 24, 2015 at 02:55:38PM -0700, Piotr Banaszkiewicz wrote:
> However, if I understand correctly, Pandas has support for
> multi-index (multi-indices?) data frames. It would take me 5 minutes
> to remove duplicate data points from the CSV files, but way longer
> to figure out how to do that in Pandas.

Or you could load the CSV with the Python stdlib, sort and dedup, and
then pass that to Pandas ;).  But still, it may not be worth the
trouble.